config.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
   // supported and will result in at most 1000 samples per second.
   // Note: `samplingRate` should never be set to 0. To disable tracing, use
```

Comment:
Not part of this CL, but, what happens when you set it to 0? Users will set it to zero despite our warning. At the least the documentation here should explain what would happen. At best, we should detect samplingRate of 0 and avoid whatever the problem is, or throw.